.github/scripts/generate_binary_build_matrix.py

```diff
@@ -41,14 +51,27 @@
 
 CONDA_CONTAINER_IMAGES = {
     **{
-        # TODO: Re-do manylinux CUDA image tagging scheme to be similar to
-        #       ROCM so we don't have to do this replacement
```

Comment:
did we fix this todo?

Reply:
this is to be addressed later, not in this PR

Reply:
hmm maybe we shouldn't remove this TODO comment then?

Reply:
oh nvm sorry, I didn't realize you were commenting on a TODO that was removed; yes we did resolve this